torchtext/prototype/t5/modules.py

```diff
@@ -0,0 +1,62 @@
+# /* Portions Copyright (c) Meta Platforms, Inc. and affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Original code is taken from
+# https://github.com/huggingface/transformers/blob/main/src/transformers/models/t5/modeling_t5.py
+# */
```

Comment:
Did we verify if this should go in the license header or in docstrings for the class itself?

Reply:
Gotcha thanks for clarifying!